src/common/zscore/cgmzscore/src/main.py

```diff
@@ -0,0 +1,103 @@
+    GROWTH_CHART[table_name] = new_table
+
+
+def z_score_calculation(lms_box_cox: object) -> int:
+    lms_box_cox.get_file_name_from_data()
+    lms_box_cox.get_lms_value(GROWTH_CHART)
+    skew, median, coff, measurement = lms_box_cox.resolve_lms_value()
+    z_score_value = Zscore(skew, median, coff, measurement).z_score_measurement()
+    return z_score_value
+
+
+def z_score_wfa(weight: str, age_in_days: str, sex: str) -> int:
+    lms_box_cox = LMS_BOX_COX('wfa', weight=weight, muac=None, age_in_days=age_in_days, sex=sex, height=None)
+    return z_score_calculation(lms_box_cox)
+
+
+def z_score_wfh(weight: str, age_in_days: str, sex: str, height: str) -> int:
+    if D(age_in_days) <= 731:
+        return z_score_wfl(weight, age_in_days, sex, height)
+    lms_box_cox = LMS_BOX_COX('wfh', weight=weight, muac=None, age_in_days=age_in_days, sex=sex, height=height)
+    return z_score_calculation(lms_box_cox)
+
+
+def z_score_wfl(weight: str, age_in_days: str, sex: str, height: str) -> int:
+    if D(age_in_days) > 731:
+        return z_score_wfh(weight, age_in_days, sex, height)
+    lms_box_cox = LMS_BOX_COX('wfl', weight=weight, muac=None, age_in_days=age_in_days, sex=sex, height=height)
+    return z_score_calculation(lms_box_cox)
+
+
+def z_score_lhfa(age_in_days: str, sex: str, height: str) -> int:
+    lms_box_cox = LMS_BOX_COX('lhfa', weight=None, muac=None, age_in_days=age_in_days, sex=sex, height=height)
+    return z_score_calculation(lms_box_cox)
+
+
+def z_score_with_class(weight: str, muac: str, age_in_days: str, sex: str, height: str):
+    wfa = z_score_wfa(weight=weight, age_in_days=age_in_days, sex=sex)
+    if wfa < -3:
+        class_wfa = 'Severely Under-weight'
+    elif -3 <= wfa < -2:
+        class_wfa = 'Moderately Under-weight'
+    else:
+        class_wfa = 'Healthy'
+
+    if D(age_in_days) > 24:
+        dummy = 'Z_score_WFH'
+        dummy_class = "Class_WFH"
+        wfl = z_score_wfh(weight=weight, age_in_days=age_in_days, sex=sex, height=height)
+    else:
+        dummy = 'Z_score_WFL'
+        dummy_class = "Class_WFL"
+        wfl = z_score_wfl(weight=weight, age_in_days=age_in_days, sex=sex, height=height)
+    class_wfl = calculate_sam_mam(weight, muac, age_in_days, sex, height)
+
+    lhfa = z_score_lhfa(age_in_days=age_in_days, sex=sex, height=height)
+    if lhfa < -3:
+        class_lhfa = 'Severely Stunted'
+    elif -3 <= lhfa < -2:
+        class_lhfa = 'Moderately Stunted'
+    else:
+        class_lhfa = 'Healthy'
+
+    return json.dumps({'Z_score_WFA': wfa, 'Class_WFA': class_wfa, dummy: wfl,
+                       dummy_class: class_wfl, 'Z_score_HFA': lhfa, 'Class_HFA': class_lhfa})
+
+
+def calculate_sam_mam(weight: str, muac: str, age_in_days: str, sex: str, height: str) -> str:
+    assert muac is not None
+    diagnosis = ''
+
+    if D(age_in_days) > 731:
+        wfl = z_score_wfh(
+            weight=weight, age_in_days=age_in_days, sex=sex, height=height)
```

Comment:
```suggestion
        wfl = z_score_wfh(weight=weight, age_in_days=age_in_days, sex=sex, height=height)
```

See more occurrences below